src/Components/Bounty/index.js

```diff
@@ -1,10 +1,66 @@
-import React from "react";
+import React, { Component } from 'react';
 import "./style.css";
+import findByType from "./../../utils/findByType"
 
 // Main Bounty component, belongs in Main Display
 
-const Bounty = () => (
-    <div className="bounty">I'm the bounty Main Component!</div>
-);
+const BountyInfo = () => null;
+const BountyReward = () => null;
+const BountyTitle = () => null;
 
+class Bounty extends Component {
+
+    renderTitle() {
+        const { children } = this.props;
+        const bountyTitle = findByType(children, BountyTitle);
```

Comment:
Are you searching the DOM for an element and then storing that as the bountyTitle??